roles/docker_cache/templates/nginx.conf.j2

```diff
@@ -53,6 +53,13 @@ http {
         listen [::]:80;
         {% endif %}
 
+        # Only allow access on STFC and private addresses
+        allow 130.246.0.0/16;
+        allow 172.16.0.0/12;
+        allow 192.168.0.0/16
```

Comment:
Oh actually I just spotted this before merging, your missing a `;` here

Reply:
So I was, fixed now